package.json

```diff
@@ -22,11 +22,12 @@
     "format": "prettier --write \"src/**/*.+(ts|json)\"",
     "test": "jest",
     "test:coverage": "jest --coverage --json --outputFile=jest.results.json && cat ./coverage/lcov.info | coveralls",
+    "test:coverage:html": "jest --silent=false --coverage --coverageReporters html",
     "codegen": "npm run graphql:codegen && npm run swagger:codegen",
     "prepare": "npm run build",
     "prepublishOnly": "npm test && npm run lint",
     "graphql:codegen": "graphql-codegen --config graphql-codegen.yml",
-    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/main/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false"
```

Comment:
@mfw78 why?

Reply:
Instead of using the `main` branch to pull from on services, this way a specific commit hash from services has its openapi.yml pulled, and therefore ensures reproducible builds for the package.

Reply:
Yes, make sense, we just thought it is convenient to use have it pointing to main, so we would for sure notice if there's changes in it and keep more in sync

The fear is that we forget to keep it up to date cause you need to actively change it there

If we really want to add something like this, i would suggest to point to a tagged version from the backend 

@shoom3301 @alfetopito WDYT?